packages/useraccounts-core/lib/methods.js

```diff
@@ -89,8 +88,14 @@ if (Meteor.isServer) {
                 }).data;
 
                 if (!apiResponse.success) {
-                    throw new Meteor.Error(403, AccountsTemplates.texts.errors.captchaVerification,
-                      apiResponse['error-codes'] ? apiResponse['error-codes'].join(", ") : "Unknown Error.");
+                    try {
+                        var msg = apiResponse['error-codes'].join(", ");
+                    } catch(e) {
+                        console.error('Exception Caught: ',e);
+                        var msg = "Unknown Error.";
+                    }
+
+                    throw new Meteor.Error(403, AccountsTemplates.texts.errors.captchaVerification, msg);
```

Comment:
what happened here? we shouldn't change this package since it's just a third party library we cloned, and we'll replace with the official version once it's released.